lib/query-sql.js

```diff
@@ -1,5 +1,5 @@
-const alasql = require('./alasql')
+const filterAndTransform = require('./filter-and-transform')
 
 module.exports = function (statement, params) {
-  return alasql(statement, params)
+  return filterAndTransform(statement, params)
 }
```

Comment:
What's the value of this file?

Reply:
Unknown.  Just part of the repo's legacy.  I'm working on a follow up PR that will change file structure and eliminate it.

Reply:
Cool, then this PR looks good to me